index.html

```diff
@@ -503,66 +504,64 @@
 
 The Web is a social and technical system made up of [=information flows=]. Because this document
 is specifically about [=privacy=] as it applies to the Web, it focuses on privacy with respect to
-information flows. Our goal is not to cover all privacy issues, but rather to provide enough
-background to support the Web community in making informed decisions about privacy and to weave
-privacy into the architecture of the Web. Few architectural principles are absolute, and privacy is
-no exception: privacy can come into tension with other desirable properties of an ethical
-architecture, and when that happens the Web community will have to work together to strike the right
-balance.
+information flows. Our goal is not to cover all possible privacy issues, but rather to provide enough
+background to support the Web community in making informed decisions about privacy and in weaving
+privacy into the architecture of the Web.
+
+Few architectural principles are absolute, and privacy is no exception: privacy can come into tension
+with other desirable properties of an ethical architecture, and when that happens the Web community
+will have to work together to strike the right balance.
 
 Information is power. It can be used to predict and to influence people, as well as to design online
 spaces that control people's behaviour. The collection and [=processing=] of information in greater
 volume, with greater precision and reliability, with increasing interoperability across a growing
 variety of data types, and at intensifying speed is leading to an unprecedented concentration of power
-that threatens private and public liberties.
-
-What's more, automation and the increasing computerisation of all aspects of our lives
-both increase the power of information and decrease the cost of a number of intrusive
+that threatens private and public liberties. What's more, automation and the increasing computerisation
+of all aspects of our lives both increase the power of information and decrease the cost of a number of intrusive
 behaviours that would be more easily kept in check if the perpetrator had to be in the same room as
 the victim.
 
-These asymmetries of information and of automation create commanding asymmetries of power.
+These asymmetries of information and of automation create powerful asymmetries of power.
```

Comment:
```suggestion
These asymmetries of information and of automation create significant asymmetries of power.
```
? To avoid repeating "power".